builds/fabric8-che/pom.xml

```diff
@@ -172,6 +190,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
To me dependency analysis is not something we need in assembly projects, and it doesn't work well with overlay. It is useful in classic jar projects.